tower-web/src/assets/variables.scss

```diff
@@ -11,12 +11,10 @@ $submitted-orange: #f57f27;
 $thin-grey: #dee2e6;
 
 $body-background: #f8f9fa;
+$sidebar-width: 22rem;
+$sidebar-collapsed-width: 4rem;
 $sidebar-background: #f8f9fa;
 $sidebar-border: 1px solid $thin-grey;
 
-$navbar-height: 64px;
+$navbar-height: 63px;
 $font-size-root: 0.90rem;
```

Comment:
Why the navbar is 1px smaller? #curiosity :D

Reply:
I think I spotted a 1px gap between the sidebar and the nav..! That was it 🤷‍♂